css-typed-om/Overview.bs

```diff
@@ -4801,6 +4806,11 @@ which will be filled in with more detail as we go.
 
 : <a>top</a>
 ::
+    1. If the specified or computed value is the ''auto'' keyword,
+        return the result of reifying the "auto" keyword.
+    2. If the specified or computed value is of type <<length>>,
+        return the result of reifying the <<length>> value.
```

Comment:
Need to link to the algo defining this (reify a numeric value). Can combine this with the next step, since lengths and %s use the same algo.